### PR TITLE
add ScriptLocalPath param set for Set-AzVmRunCommand

### PR DESCRIPTION
**Pull request by grizzlytheodore**
## Description
addressing https://github.com/Azure/azure-powershell/issues/19468

## Checklist

- [x] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

**Comment by grizzlytheodore** [comment]
how do i update changelog when updating generation branch? @VeryEarly

**Comment by vidai-msft** [comment]
@grizzlytheodore changelog can be added from generation to main sync. You can let me know and I will help add.

File: src/Compute/Compute.Autorest/custom/Set-AzVMRunCommand_ScriptLocalPath.ps1 [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Code generated by Microsoft (R) AutoRest Code Generator.Changes may cause incorrect behavior and will be lost if the code
+# is regenerated.
+# ----------------------------------------------------------------------------------
+
+function Set-AzVMRunCommand_ScriptLocalPath {
+    [OutputType([Microsoft.Azure.PowerShell.Cmdlets.Compute.Models.Api20210701.IVirtualMachineRunCommand])]
+    [CmdletBinding(PositionalBinding=$false, SupportsShouldProcess, ConfirmImpact='Medium')]
```
Comment (vidai-msft) [inline src/Compute/Compute.Autorest/custom/Set-AzVMRunCommand_ScriptLocalPath.ps1:19]: Is the DefaultParameterSetName missing here?

Comment (grizzlytheodore) [reply src/Compute/Compute.Autorest/custom/Set-AzVMRunCommand_ScriptLocalPath.ps1:19]: do i need to set that? looking at other customized cmdlets in the folder, it doesn't seem necessary. Looking at the doc, it looks like it resolves the original as the defaultParameterSet by default. 
https://github.com/Azure/azure-powershell/pull/19935/files#diff-98b2e209fe660fbafe42a698f08f3105e161521fdca07891934035f26f317b01R15

**Comment by grizzlytheodore** [comment]
@vidai-msft  I want to add 
"Added 'ScriptFilePath' parameter sets for `Set-AzVMRunCommand` and `Set-AzVmssVMRunCommand` to allow users to pass in the path of the file that has the run command script."

Please show me where I add to add that for changelog

**Comment by VeryEarly** [comment]
> @vidai-msft I want to add "Added 'ScriptFilePath' parameter sets for `Set-AzVMRunCommand` and `Set-AzVmssVMRunCommand` to allow users to pass in the path of the file that has the run command script."
> 
> Please show me where I add to add that for changelog

Hi @grizzlytheodore ,

Please leave changelog here in description of this PR, I'll add it to release branch while migration.

**Comment by grizzlytheodore** [comment]
its in the comment you just reply to @VeryEarly

**Review by VeryEarly — APPROVED** [review]

**Pull request merged**